src/coreclr/jit/block.cpp

```diff
@@ -835,10 +837,61 @@ void BasicBlock::CopyTarget(Compiler* compiler, const BasicBlock* from)
         case BBJ_COND:
             SetCond(from->GetTrueTarget());
             break;
+        case BBJ_ALWAYS:
+            SetKindAndTarget(from->GetKind(), from->GetTarget());
+            CopyFlags(from, BBF_NONE_QUIRK);
```

Comment:
Just a question for my own understanding: In situations like this where we can fall through into the next case, is it preferable to fall through or to duplicate a small amount of code for readability? I've seen both patterns leveraged in the JIT.

Reply:
I don't think there's a required pattern. I could have done the `CopyFlags` then fall through. In fact, the `CopyFlags` would be fine for other block kinds as well, so we could just do it unconditionally, but I chose to make it only happen for BBJ_ALWAYS to be clear that's the only kind where that flag should be set.